pytition/petition/views.py

```diff
@@ -16,8 +16,8 @@
 from django.contrib.auth.models import User
 from django.contrib.auth.forms import PasswordChangeForm
 from django.urls import reverse
-from django.utils.text import slugify
 
+from django.utils.text import slugify
```

Comment:
This line move is not necessary I suppose.

Reply:
oups you're right, sorry